src/Workspaces/Core/Portable/Shared/Extensions/IMethodSymbolExtensions.cs

```diff
@@ -267,98 +264,5 @@ public static IMethodSymbol RenameTypeParameters(this IMethodSymbol method, Immu
             p2 = method2.OriginalDefinition.Parameters;
             return p1.Select(p => p.Type).ToList().AreMoreSpecificThan(p2.Select(p => p.Type).ToList());
         }
-
-        public static bool TryGetPredefinedComparisonOperator(this IMethodSymbol symbol, out PredefinedOperator op)
```

Comment:
Same here - deleted code moved into a separate file in shared layer. Same holds for other symbol extension file edits after this file.